components/table/table-col-sort-button.js

```diff
@@ -40,13 +40,26 @@ export class TableColSortButton extends LocalizeCoreElement(RtlMixin(FocusMixin(
 				reflect: true,
 				type: String
 			},
+			/**
+			 * Custom message to define the type of sorting that's occuring.
+			 * @type {String}
+			 */
+			description: {
+				reflect: true,
+				type: String
+			},
```

Comment:
So I believe we want to pull the text for the `aria-description` for the dropdown button directly from the selected item in the menu and don't want to have the consumer have to specify it.